content/en/serverless/insights/_index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Serverless Insights
+kind: documentation
+---
+
+{{< img src="serverless/serverless_actionable_insights_detail.png" alt="Auto-generated insights for your serverless applications" >}}
+
+Datadog automatically generates suggestions to resolve errors, performance problems, and optimize cost for your serverless applications.
+
+## Setup
+
+Datadog uses AWS CloudWatch metrics, Datadog enhanced AWS Lambda metrics, and Lambda `REPORT` logs to suggest insights to you. To set these up,
```

Comment:
Will everyone know what `REPORT` logs are or do we need to explain that?

Reply:
a link out would be good here